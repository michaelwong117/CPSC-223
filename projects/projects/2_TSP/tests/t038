#!/bin/bash
# 256 cities, exchange any

trap "/usr/bin/killall -q -u $USER ./TSP 2>/dev/null" 0 1 2 3 9 15
trap "/bin/rm -f $STDERR" 0 1 2 3 9 15
if [ ! -x ./TSP ]; then
  echo './TSP is missing or not executable'
  echo './TSP is missing or not executable' 1>&2
  exit 1
fi


/c/cs474/bin/run -stderr=/dev/null ./TSP /c/cs223/hw2/Tests/t038.in -exchange any < /dev/null
