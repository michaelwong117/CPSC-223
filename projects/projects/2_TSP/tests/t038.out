-exchange any     :    404656.00 RFD CGX TDZ FDY AXV MUT 1G4 LAS DWA A87 KHV MZG TOD KUA RRG AMP DUR NLA PNR LBQ BSG KGS SLS TGM KSC YXZ TVC RAC HPT ENV SUU APC A61 KWJ FOC HZH DIG GAU BIR PMI MJV PHI GRP PET URG MBK MBZ TFF LOI MLD LAR MCI JEF BRY IAD OCE BGI BMJ BVB GRA CZE CAP CCC CXY DUB CEG SXL HZL THV IDI JFN BCB INT SAV UST TIX OAJ BID BBF ME3 WIC HDF OSP VST WFK MLT DXR VGC PHN PTK WBR MGC SIK UOX MS1 ELA PDS AZ3 YUM UDD RBF BJU PGH DEL PLZ CPT USH ZCO BHI RSA AFA CPO AVX PCE A56 A69 WSZ DUD OOM XRH PQQ ARM IFL CGC TKK HND ITM UKB IZO CIH LHW ULN TJM VLL SCQ HUH CHT RDA MTD BXU KKI UXM SKW AK3 A53 LAK GLN BOY LRL LUE ILZ 04A A94 NCL ZXE FKB FRF BYU KLU OMO OHD NAP CIA LUG SNR CFR BBS BHX GOH ZPB YGX BMA GNA KIV ERD SZF RAE TIE MMQ AMH MSH MWD NBC CA4 HSV HAB MS3 GUF VWT THA K01 CCY JMN EVM YOD YZH YOP FQJ 05A A07 HGZ TAL PAQ PGM A03 KLL SCM LAO SBW CAH BKK MOG YSP OSU LNP ARE VQS SXM BXE BYK DIV GGN KBS CMG TUC CJC TCQ PCG GDL LTO NKX MMH LKV SMN VWW WA7 PWT SQV FRD DTR CXH PTR ORI NKI GFA CBF ZBX FYV MWL PUB CNE ILL MIC MSP AUM PCD RFD
